### PR TITLE
Fixed text-align: right for rtl languages

### LINKED ISSUE #8771: Library Search - Category labels are not right aligned when viewed in RTL language
## Observed behavior
The labels of the Categories are not right aligned when viewed in RTL language.

## Expected behavior
The labels of the Categories should be right aligned.

## Steps to reproduce the issue
1. Install the following [build.](https://github.com/learningequality/kolibri/releases/tag/v0.15.0-beta2)
2. Import some channels and run the following management command: `kolibri manage labeltestdata `
3. Switch to RTL language such as Arabic and observe the labels of the Categories.

## Usage Details
 - OS: Windows 10
 - Browser: Chrome

### PR DESCRIPTION
**Pull request by akash5100**
## Summary

Fixed text-align: right for RTL languages, a problem reported by @pcenov
* Added a condition to align text depending on value of RTL or LTR

## References

closes #8771
before | after
---------|-------
Screenshot from 2021-12-01 17-43-33 | Screenshot from 2021-12-01 17-05-10

* no changes to LTR languages

Screenshot from 2021-12-01 17-04-36

----

## Testing checklist

- [x] Contributor has fully tested the PR manually
- [x] If there are any front-end changes, before/after screenshots are included
- [ ] Critical user journeys are covered by Gherkin stories
- [ ] Critical and brittle code paths are covered by unit tests

## PR process

- [ ] PR has the correct target branch and milestone
- [ ] PR has 'needs review' or 'work-in-progress' label
- [ ] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')
- [ ] If this is an important user-facing change, PR or related issue has a 'changelog' label
- [ ] If this includes an internal dependency change, a link to the diff is provided

## Reviewer checklist

- Automated test coverage is satisfactory
- PR is fully functional
- PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- External dependency files were updated if necessary (`yarn` and `pip`)
- Documentation is updated
- Contributor is in AUTHORS.md

### CODE REVIEW

**Comment by MisRob** [comment]
Thank you @akash5100, changes look good to me. It seems that we merged some updates of the same file recently, would you like to resolve conflicts as well? Then we will see if all automatic checks are passing and I will test the PR locally.

If you don't get to it, I can do it later. In any case, thank you for your contribution.

**Review by akash5100 — COMMENTED** [review]
I had a merge conflict because of `'text-transform': 'capitalize',`  on the same line, So I accepted the upcoming changes and marked it as resolved.

**Comment by MisRob** [comment]
I believe that the conflict was coming from #8791 where we removed `'capitalize'` in favor of `'none'`, so let's keep the new value there to not cause regressions. The only change this PR needs to have is your update of the '`text-align`' value.

File: kolibri/plugins/learn/assets/src/views/EmbeddedSidePanel/index.vue [hunk 1]
```diff
@@ -263,7 +263,9 @@
           color: this.$themeTokens.text,
           width: '100%',
           border: '2px solid transparent',
-          'text-align': 'left',
+          'text-transform': 'capitalize',
```
Comment (rtibbles) [inline kolibri/plugins/learn/assets/src/views/EmbeddedSidePanel/index.vue]: Removing this line will be the correct merge conflict resolution and will fix your linting error.
```suggestion
```

**Review by MisRob — APPROVED** [review]
Everything's working as expected. Thank you, @akash5100

**Pull request merged**